ingestion.go

```diff
@@ -245,10 +245,24 @@ func (p *PeopleProperties) SetIp(ip net.IP) {
 	p.Properties[string(PeopleGeolocationByIpProperty)] = ip.String()
 }
 
+// Note: if no ip is provided, we will not track by default
```

Comment:
what is the config way to track caller-ip by default?

Reply:
We do on client-side SDK's but default, but it makes more sense since it's the client's IP address. Since this is server-side it will probably all be the same IP which doesn't help

Reply:
my ask is I don't see a way for the client to set track user profile's geolocation based on IP seen by our API edge server.
It's either totally no ip-Tracking or must explicitly set IP. 
I know we assume this SDK is mostly used server-side, and that's fine to set no-tarcking-IP by default. However, there should be a way to turn on that behavior.